JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,23 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const minutesFromSeconds = seconds / 60;
+  const hoursFromMinutes = Math.floor(minutesFromSeconds / 60);
+  const leftMinutes = Math.floor(minutesFromSeconds - (hoursFromMinutes * 60));
+  const leftSeconds = seconds - (hoursFromMinutes * 60 * 60) - (leftMinutes * 60);
```

Comment:
We can get the left seconds from the remainder operator:

```suggestion
  const leftSeconds = seconds % 60;
```

Reply:
Refactored ✅